mobile/screens/AddBook.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import { View } from "react-native";
+import { StyleSheet, Image, TouchableOpacity, Alert } from "react-native";
+import AddBookForm from "./AddBookForm";
+import { Portal, Modal } from "react-native-paper";
+
+export default function AddBook(props) {
+  const [visible, setVisible] = useState(false);
+
+  const toggleOverlay = () => {
+    setVisible(!visible);
+  };
```

Comment:
This needs to be wrapped in a `useCallback` hook to prevent unnecessary rerenders. I'll change it

Reply:
@taneliang I forgot to remove this, this is not being used